src/rtc.c

```diff
@@ -0,0 +1,792 @@
+/**
+ * @file rtc.c
+ * @brief Real-Time Clock Subsystem
+ * @ingroup rtc
+ */
+
+#include <string.h>
+#include <time.h>
+#include "libdragon.h"
+#include "system.h"
+
+/**
+ * @defgroup rtc Real-Time Clock Subsystem
+ * @ingroup libdragon
+ * @brief Real-time clock interface.
+ *
+ * The Joybus real-time clock is a cartridge peripheral that uses a battery
+ * to power a clock that tracks the date, time, and day of the week. The
+ * real-time clock keeps running even when the N64 is powered-off. The
+ * Joybus RTC is accessed through the serial interface (SI) similar to EEPROM
+ * and controllers. The Joybus RTC was only ever available on one official
+ * cartridge that was only available in Japan: Dōbutsu no Mori (Animal Forest).
+ * There is also a real-time clock included in the N64DD hardware, which uses
+ * a different interface and is not currently supported by libdragon.
+ *
+ * To check if the real-time clock is available, call #rtc_init.
+ * To read the current time from the real-time clock, call #rtc_get.
+ * To check if the real-time clock supports writes, call #rtc_is_writable.
+ * To write a new time to the real-time clock, call #rtc_set.
```

Comment:
I would mention here that you can call `time(NULL)` to get the current time. Something like `call #rtc_get or time(NULL)`.

Reply:
Fixed in https://github.com/DragonMinded/libdragon/pull/152/commits/544baa50639e55196c93c17c97b8cda1365f7bfc